tusclient/request.py

```diff
@@ -99,7 +101,10 @@ async def perform(self):
         self.add_checksum(chunk)
         try:
             async with aiohttp.ClientSession(loop=self.io_loop) as session:
-                async with session.patch(self._url, data=chunk, headers=self._request_headers) as resp:
+                ssl = None if self.verify else False
+                async with session.patch(
+                        self._url, data=chunk, headers=self._request_headers,
+                        ssl=ssl) as resp:
```

Comment:
Does this entirely disable SSL or only disable verifying the certificate? It looks more like the first one to me.

Reply:
`ssl` refers to SSL validation mode [per aiohttp docs](https://docs.aiohttp.org/en/stable/client_reference.html?highlight=ssl#aiohttp.ClientSession):

> ssl –
> SSL validation mode. None for default SSL check
>     (ssl.create_default_context() is used), False for skip SSL certificate validation, aiohttp.Fingerprint for fingerprint validation, ssl.SSLContext for custom SSL certificate validation.
>    Supersedes verify_ssl, ssl_context and fingerprint parameters.

There's a more descriptively named `verify_ssl` argument, but that was deprecated in favor of `ssl`.